rails/app/assets/images/thumbtack-not-pinned.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><style>svg{fill:#a6a6a6}</style><path d="M32 32C32 14.3 46.3 0 64 0H320c17.7 0 32 14.3 32 32s-14.3 32-32 32H290.5l11.4 148.2c36.7 19.9 65.7 53.2 79.5 94.7l1 3c3.3 9.8 1.6 20.5-4.4 28.8s-15.7 13.3-26 13.3H32c-10.3 0-19.9-4.9-26-13.3s-7.7-19.1-4.4-28.8l1-3c13.8-41.5 42.8-74.8 79.5-94.7L93.5 64H64C46.3 64 32 49.7 32 32zM160 384h64v96c0 17.7-14.3 32-32 32s-32-14.3-32-32V384z"/></svg>
```

Comment:
I know the issue stated to use "font-awesome" — which is our bad on leaving that in when we've since removed font-awesome 😅. I'm wondering if we would benefit from switching to using icons from Google's open source [Material Icons](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Apush_pin%3AFILL%401%3Bwght%40400%3BGRAD%400%3Bopsz%4048) which has supplied several of our other icons in recent icon additions.

This isn't blocking.

Reply:
No problem, I updated to use google icon svg